src/components/ConfigureGame/ConfigureGame.css

```diff
@@ -3,37 +3,37 @@
   flex-direction: column;
 }
 
-.configure-screen__name-input-wrapper {
+.player__wrapper {
```

Comment:
If you're defining a new scope, it should be based with just `.player` no `__wrapper`